scripts/skills/perks/perk_rf_offhand_training.nut

```diff
@@ -1,7 +1,6 @@
 this.perk_rf_offhand_training <- ::inherit("scripts/skills/skill", {
 	m = {
 		StaminaModifierThreshold = -10,
-		IsSpent = true,
```

Comment:
I prefer the term `IsSpent` for the only variable enforcing the main behavior of this skill.
It makes the code easier to parse and is already used for similar effects in other skills plenty of times
e.g.
`return this.m.IsSpent`
or 
```
if (this.m.IsSpent)
  return;
```